assets/base.css

```diff
@@ -594,6 +594,7 @@ details > * {
   --duration-medium: 300ms;
   --duration-long: 500ms;
   --duration-extra-long: 600ms;
+  --duration-progress-bar: 750ms;
```

Comment:
Can we name this something more generic? It may be reused in the future not only for the progress bar

Something like `--duration-extra-longer` I will think of other names too